UNFLoader/Makefile

```diff
@@ -7,7 +8,13 @@ CODEFILES=main.cpp helper.cpp device.cpp debug.cpp \
 LIBFILES=Include/lodepng.c
 
 CC=g++
-DEPENDENCIES := -lncurses -lftd2xx -lpthread -lrt
+
+ifeq ($(OS_NAME),Darwin)
+	DEPENDENCIES := -lncurses -lftd2xx -lpthread
```

Comment:
macOS doesn't [seem](https://stackoverflow.com/questions/47703239/ld-library-not-found-for-lrt) [to](https://github.com/yandex/faster-rnnlm/issues/5) [need](https://github.com/lh3/bwa/issues/173) `-lrt` for building?